rhel/7.5/aws/DCOS-1.13.3/docker-18.09.2-ce/selinux_disabled/packer.json

```diff
@@ -0,0 +1,169 @@
+{
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+    "region":         "us-west-2"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "instance_type": "m4.xlarge",
+      "source_ami": "ami-036affea69a1101c9",
```

Comment:
This seems to be a wrong AMI ID. The one for RHEL 7.5 is ami-28e07e50

Reply:
Changed!